app/models/katello/host/subscription_facet.rb

```diff
@@ -277,7 +275,12 @@ def candlepin_consumer
       end
 
       def backend_update_needed?
-        %w(release_version service_level autoheal).each do |method|
+        if @flag_update
```

Comment:
Wow. I was experimenting and I just don't think there's an easy way to track changes in an association the way we want to.

Reply:
Yes, I couldn't think of a good way to do it, you would think rails could!  The only "easy" option would make add-ons serializable db column, and it would show up as a changed value.